Downloading libraries using maven failed

# Overview

Downloading libraries using maven failed.

```log
$ mvn compile
[WARNING]
[WARNING] Some problems were encountered while building the effective settings
[WARNING] 'servers.server.id' must be unique but found duplicate server with id personium.io @ C:\Program Files\apache-maven-3.6.1\conf\settings.xml
[WARNING]
[INFO] Scanning for projects...
[INFO] 
[INFO] --------------------< io.personium:personium-core >---------------------
[INFO] Building personium-core Maven Webapp 1.7.21-SNAPSHOT
[INFO] --------------------------------[ war ]---------------------------------
Downloading from central: http://repo1.maven.org/maven2/org/glassfish/jersey/containers/jersey-container-servlet/2.29.1/jersey-container-servlet-2.29.1.pom
Downloading from central: http://repo1.maven.org/maven2/org/glassfish/jersey/inject/jersey-hk2/2.29.1/jersey-hk2-2.29.1.pom
Downloading from central: http://repo1.maven.org/maven2/org/glassfish/jersey/media/jersey-media-moxy/2.29.1/jersey-media-moxy-2.29.1.pom
Downloading from central: http://repo1.maven.org/maven2/org/glassfish/jersey/test-framework/providers/jersey-test-framework-provider-grizzly2/2.29.1/jersey-test-framework-provider-grizzly2-2.29.1.pom
Downloading from central: http://repo1.maven.org/maven2/org/glassfish/jersey/test-framework/jersey-test-framework-core/2.29.1/jersey-test-framework-core-2.29.1.pom
Downloading from central: http://repo1.maven.org/maven2/org/glassfish/jersey/containers/jersey-container-grizzly2-servlet/2.29.1/jersey-container-grizzly2-servlet-2.29.1.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.208 s
[INFO] Finished at: 2020-01-23T10:58:16+09:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project personium-core: Could not resolve dependencies for project io.personium:personium-core:war:1.7.21-SNAPSHOT: Failed to collect dependencies at org.glassfish.jersey.containers:jersey-container-servlet:jar:2.29.1: Failed to read artifact descriptor 
for org.glassfish.jersey.containers:jersey-container-servlet:jar:2.29.1: Could not transfer artifact org.glassfish.jersey.containers:jersey-container-servlet:pom:2.29.1 from/to central (http://repo1.maven.org/maven2): Failed to transfer file http://repo1.maven.org/maven2/org/glassfish/jersey/containers/jersey-container-servlet/2.29.1/jersey-container-servlet-2.29.1.pom with status 
code 501 -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following 
articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

# Cause

January 15, 2020, The Central Repository no longer supports insecure communication over plain HTTP and requires that all requests to the repository are encrypted over HTTPS.

https://support.sonatype.com/hc/en-us/articles/360041287334

--- a/pom.xml
+++ b/pom.xml
@@ -281,7 +281,7 @@
             <id>central</id>
             <name>Maven Repository Switchboard</name>
             <layout>default</layout>
-            <url>http://repo1.maven.org/maven2</url>
+            <url>https://repo1.maven.org/maven2</url>
             <snapshots>
                 <enabled>false</enabled>
             </snapshots>

